admob/src/include/firebase/admob/types.h

```diff
@@ -17,7 +17,9 @@
 #ifndef FIREBASE_ADMOB_SRC_INCLUDE_FIREBASE_ADMOB_TYPES_H_
 #define FIREBASE_ADMOB_SRC_INCLUDE_FIREBASE_ADMOB_TYPES_H_
 
+#include <map>
 #include <string>
+#include <unordered_set>
```

Comment:
I'm sure it's fine, but just make sure both Android stlport variants that we still support have std::unordered_set.

Reply:
I didn't realize that we still support stlport.  Our [release notes say that we don't](https://firebase.google.com/support/release-notes/cpp-relnotes#version_800_-_june_18_2021).

Which versions should I be checking?

Reply:
Or did you mean that we should ensure that it's in libc++?  Looks like it was [part of the intiial port](https://github.com/llvm/llvm-project/blame/d5db71d19f11d7c31257066aea6bd41ef04f28b7/libcxx/include/unordered_set).

Reply:
gnustl, I think.

Reply:
(basically, just run cpp packaging on this and see if both Android STL versions build.)